scRNA-seq/03-normalizing_scRNA.Rmd

```diff
@@ -17,7 +17,7 @@ This notebook will demonstrate how to:
 
 ---
 
-In this notebook, we'll continue with processing the same dataset that we have been working with, moving onto normalization of scRNA-seq count data that we have already done quality-control analyses of.
+In this notebook, we'll continue with processing the same dataset that we have been working with, moving onto normalization of scRNA-seq count data that we have already done quality-control analyses on.
```

Comment:
Reading this sentence, it seems redundant. Maybe just getting rid of the last clause is the move?

```suggestion
In this notebook, we'll continue with processing the same dataset that we have been working with, moving on to normalization of scRNA-seq count data.
```